src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: $grid-gap;
+  justify-content: center;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media screen and (min-width: 1024px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(4, $card-width);
   }
 }
```

Comment:
would be better to use scss `@each` here

Reply:
Hi, sorry, not sure how to implement @each here. I thought media queries have to be through the @media function